docbook/reference/en/en-US/modules/Deprecated_security_modules.xml

```diff
@@ -0,0 +1,18 @@
+<chapter id="Deprecated_security_modules">
+    <title>Deprecated Security Modules</title>
+
+    <para>
+        These security modules are deprecated since 3.1.3.Final and users should use Keycloak provided by JBoss community instead:
```

Comment:
How about the following line instead? I'd rather not mentioning the version this is being deprecated in (it's basically a mess when you make releases)

"The following security related modules are deprecated and meant to be removed in the future:"

I would also move the reference to Keycloak after the list, in place of the paragraph on the directory structure, which is not that relevant after all.